packages/terra-form-field/docs/MigrationGuide.md

```diff
@@ -0,0 +1,76 @@
+# Migrating from terra-form's Field Component to terra-form-field
+
+The following sections describe how to migrate from the Field component provided in the terra-form package to the Field component provided in the terra-form-field package. Terra-form-field provides enhanced API and layout to begin the first phase of the OSC uplift.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-form-field`
+
+## Prop Changes
+#### Props Added
+- `errorIcon` - icon to indicates the Field is invalid.
+- `hideRequired` - indicates a Field is required, but hides the required indicator.
+- `isInvalid` - indicates whether or not a field is valid or invalid.
```

Comment:
I think can be made a little simpler.

`indicates whether or not a field is invalid`